### ISSUE TITLE
blockElytraBreakBlug in GameFixes makes it so players cannot break blocks while underwater

### ISSUE THREAD

**Issue by sepiatonal** [DevotedMC/SimpleAdminHacks#16]
This is because currently, the way the fix works is by simply stopping players from breaking blocks while their head is not in AIR. I'm not familiar enough with the thing that the fix is actually meant to fix, to figure out a solution that doesn't prevent normal breaking in water. We're just handling it by not using that fix, but it should probably be looked at in case anybody ever wants to use the fix and does not want their players to be prevented from breaking blocks while underwater (water-covered-vault new meta).

**Comment by ProgrammerDan** [comment]
Good catch. Thought we had already addressed this but must have been only
in my dreams. Thank you!
